cpp/include/cudf/scalar/scalar.hpp

```diff
@@ -50,7 +50,7 @@ class scalar {
   scalar& operator=(scalar&& other) = delete;
 
   /**
-   * @brief Returns the scalar's logical value type.
+   * @briefreturn{ the scalar's logical value type.}
```

Comment:
The previous style read like an English sentence. I have a strong preference for readability and minimal custom syntax (`@briefreturn` can't be found in any Doxygen documentation).

Is a `@return` required? I'd rather see two explicit lines of docs for `@brief` and `@return`.

Reply:
`briefreturn` is a custom alias defined in Doxyfile. We can name it anything.
Generally, I don't prefer duplicating exact same text/code. In future, these 2 texts could diverge if not taken care.
If we prefer typing the same text again, it's doable.

Reply:
I wanted to present this solution before choosing to make verbose change to all missing `@return` docs.

Reply:
added docs manually.